data_structures/treap.cpp

```diff
@@ -0,0 +1,233 @@
+
+/**
+ * @file
+ * @brief A balanced binary search tree (BST) on the basis of binary search tree
+ * and heap: the [Treap](https://en.wikipedia.org/wiki/Treap) algorithm
+ * implementation
```

Comment:
```suggestion
 * implementation
 *
```
there needs to be a space here i think, because the website doesnt render the @details tag